src/language/__tests__/parser-test.ts

```diff
@@ -224,6 +228,196 @@ describe('Parser', () => {
     ).to.not.throw();
   });
 
+  it('parses required field', () => {
+    const result = parse(
+      dedent`
+      query {
```

Comment:
Since you are testing fields specifically, please use the shorter form without operation prefix:
```
{ 
   requiredField!
}
```